.github/dependabot.yml

```diff
@@ -1,7 +1,91 @@
 version: 2
 updates:
 
-- package-ecosystem: "github-actions"
-  directory: "/"
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: daily
+- package-ecosystem: npm
+  directory: gh-actions/appauth
+  schedule:
+    interval: daily
+  groups:
+    eslint:
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: npm
+  directory: gh-actions/dispatch
+  schedule:
+    interval: daily
+  groups:
+    eslint:
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: npm
+  directory: gh-actions/jq
+  schedule:
+    interval: daily
+  groups:
+    eslint:
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: npm
+  directory: gh-actions/hashfiles
+  schedule:
+    interval: daily
+  groups:
+    eslint:
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: npm
+  directory: gh-actions/retest
+  schedule:
+    interval: daily
+  groups:
+    eslint:
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: npm
+  directory: gh-actions/torun
+  schedule:
+    interval: daily
+  groups:
+    eslint:
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: npm
+  directory: gh-actions/github/checks
+  schedule:
+    interval: daily
+  groups:
+    eslint:
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: npm
+  directory: gh-actions/github/mutex
+  schedule:
+    interval: daily
+  groups:
+    eslint:
+      patterns:
+        - "@typescript-eslint/*"
+        - "eslint*"
+        - "prettier"
+- package-ecosystem: pip
+  directory: bazel
```

Comment:
this one possibly should be bazel/website/requirements.txt

bazel/requirements.txt is empty and should probs be removed

Reply:
seems to be doing the right thing anyway